mobile/lib/features/wallet/receive_screen.dart

```diff
@@ -46,15 +46,15 @@ class _ReceiveScreenState extends State<ReceiveScreen> {
               child: const Text("Create Invoice")),
           SelectableText("Invoice: $invoice"),
           ElevatedButton(
-              onPressed: () async {
-                try {
-                  await api.openChannel();
-                  FLog.info(text: "Open Channel successfully started.");
-                } catch (error) {
-                  FLog.error(text: "Error: $error", exception: error);
-                }
-              },
-              child: const Text("Open Channel!"))
+            onPressed: () async {
+              try {
+                await api.openChannel();
```

Comment:
I think this call should be in the `WalletService` as well.

Reply:
Can move this. Shall I rename WalletInfoService to WalletService as opening a channel is not really getting info from the wallet?